.circleci/config.yml

```diff
@@ -10,7 +10,14 @@ jobs:
       - run: make test|xcpretty -r junit
       - store_test_results:
           path: build/reports/
+      - restore_cache:
+          keys:
+            - cocoapods-{{ .Environment.CIRCLE_JOB }}
       - run: pod lib lint
+      - save_cache:
+          key: cocoapods-{{ .Environment.CIRCLE_JOB }}
```

Comment:
Why does this need to be keyed by the circle job? Why not just `key: cocoapods`?

Reply:
I was worried that conflicts might occur if the same key is used for two jobs.
I don't know whether that is required or not.

Reply:
I changed the key to `cocoapods`, but it seems to be okay. 👍